app-emulation/FEX/files/FEX-2409-system-rootfs-path.patch

```diff
@@ -0,0 +1,131 @@
+diff --git a/Source/Tools/FEXQonfig/Main.cpp b/Source/Tools/FEXQonfig/Main.cpp
```

Comment:
We should only apply this if the rootfs package gets pulled in. Either behind the systemd use flag or a system-rootfs one that REQUIRED_USEs systemd

Reply:
No, this is safe to always apply, the patch does not require any specific system support, and only adds a separate directory to the default rootfs list. If it does not exist, it does nothing.